test/core/core_texture.cpp

```diff
@@ -372,7 +371,7 @@ namespace perf_generic_creation
 		}
 
 		std::clock_t TimeEnd = std::clock();
-		printf("Generic texture creation performance test: %d\n", TimeEnd - TimeBegin);
```

Comment:
A matter of opinion, but `std::cout` is a little more flexible when converting data to text and `std::endl` adds platform-specific newline symbols.